### ISSUE TITLE
Domain props does not correctly apply zoom_level after page crash

### ISSUE THREAD

**Issue by aidanholm** [luakit/luakit#357]
Steps to reproduce:
1. Set `globals.domain_props.all.view_level = 1.4`: pages are zoomed in.
2. Run `:lua w.view:crash()` to test crash the page. (Note: might also crash other tabs, depending on your process limit).
3. Crash page no longer zoomed in, subsequent pages viewed on this tab are not zoomed in either. 

cc @xkpd3

**Comment by aidanholm** [comment]
Quick fix, in `lib/domain_props.lua`:
```lua
        for k, prop in pairs(props.props) do
            msg.info("domain prop: %s = %s (%s)", k, prop, props.domain)
+           if k == "zoom_level" then view.zoom_level = 1.0 end
            view[k] = prop
        end
```

**Comment by aidanholm** [comment]
Fixed in C at dafca7b0
